sympy/matrices/sparse.py

```diff
@@ -463,15 +463,6 @@ def _eval_has(self, *patterns):
             zhas = False
         return any(self[key].has(*patterns) for key in self._smat) or zhas
 
-    def _eval_is_Identity(self):
-        if not all(self[i, i] == 1 for i in range(self.rows)):
-            return False
-        return len(self._smat) == self.rows
-
-    def _eval_is_symmetric(self, simpfunc):
-        diff = (self - self.T).applyfunc(simpfunc)
-        return len(diff.values()) == 0
```

Comment:
The judgements from sparsity alone can make false-positives for symbolic matrices, so I have removed them for now.